src/testdir/test_vim9_class.vim

```diff
@@ -101,13 +101,13 @@ def Test_class_basic()
       this. count
     endclass
   END
-  v9.CheckSourceFailure(lines, 'E1317: Invalid object variable declaration: this. count', 3)
+  v9.CheckSourceFailure(lines, 'E1318: Not a valid command in a class: this. count', 3)
 
   # Use "that" instead of "this"
   lines =<< trim END
     vim9script
     class Something
-      this.count: number
+      var count: number
```

Comment:
Can you add a test for using the previous syntax using the "this" prefix with object and class variables and check whether they all fail?